### DISCUSSION TITLE
Unable to create an account - no confirmation email

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#7433] [General]
Hi. I created an account on the Chromiecraft website but never received a confirmation email. I tried creating it with an alias as was recommended in #2394 but to no avail, I also did not receive a confirmation email. I made sure to check every folder in my inbox, spam, trash etc, but there is nothing anywhere.

**Comment by Helias** [comment]
Hi, I've just created a new account and the email arrived.
I think that I got what are your emails and retries, and I don't see any potential issues on them, it's weird that the emails didn't arrive to you.
If you contact me on Discord in DM or open a support ticket I could manage this better, anyways, I sent you some reset password email, you should have received it

**Comment by ghost** [comment]
Ah, so sorry for the confusion. Turns out that the emails were, after all, in the spam folder. The 3rd party client I was using didn't show them in there...

**Reply by Helias** [reply]
no problem! I am happy that you solved ;-)
